templates/crafty.service.j2

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=Crafty
+After=network-online.target
+
+[Service]
+Type=simple
+ExecStart={{ CRAFTY_HOME }}/venvs/crafty/bin/matterbot
+Restart=always
+RestartSec=10
+WorkingDirectory={{ CRAFTY_HOME }}/crafty
+User=crafty
+Group=crafty
+Envrionment="MATTERMOST_BOT_URL={{ CRAFTY_MATTERMOST_URL }}"
```

Comment:
@smarnach Typo -- `Environment`?

Reply:
@UmanShahzad Yeah, that's what I meant.  Hey, I'm still working on this. :)

Reply:
:p